README.txt

```diff
@@ -35,26 +35,46 @@ found at
 
 The resulting image is the same image used as the base for software
 installation from NI's tools. An NI installation also adds several
-more packages of NI's own software. NI's additions include scripts and
-other software affecting login. To enable login without NI software, you
-will need to make a few more changes. Assuming you are in safe mode and
-the run mode filesystem is mounted at /mnt/userfs:
+more packages of NI's own software.
+
+** NOTE **
+Building packages through OpenEmbedded can use significant disk space,
+on the order of tens of gigabytes. If you are preparing a virtual machine
+to build images, make sure to allocate sufficient disk space.
+
+The kernel image created by the build system cannot be used.  If you
```

Comment:
This is not strictly correct for all hardware. The kernel would be usable from an x64 build. Adjust the verbiage.